src/violations.rs

```diff
@@ -4253,6 +4253,20 @@ impl AlwaysAutofixableViolation for EndsInPeriod {
     }
 }
 
+define_violation!(
+    pub struct UseImperativeMood(pub String);
+);
+impl Violation for UseImperativeMood {
+    fn message(&self) -> String {
+        let UseImperativeMood(first_line) = self;
+        format!("First line of docstring should be in imperative mood: \"{first_line}\"")
```

Comment:
I think using a raw string literal `r#" ... "#` is nicer when the string contains double quotes.

Reply:
I followed the style used elsewhere in the file.